### PR TITLE
Implement breaking svelte-kit changes

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Implement the breaking changes necessary to upgrade svelte-kit. Main focus is the new routes structure. Check https://kit.svelte.dev/docs/introduction for info.

Also, `svelte-kit` now requires `vite` as a separate dependency. This fixes that.

Moved the API_URL building logic to `vite.config.ts` also.

I also noticed that the LICENSE file was not matching to the license in `frontend`. AFAIK we should be using MPL license, which saves us from people copying the design but still gives us zero responsibility in terms of warranty and zero liability.

### CODE REVIEW

**Review by ZePedroResende — APPROVED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -1,4 +1,3 @@
-nodejs 16.14.0
+nodejs 16.17.0
 elixir 1.13.3
 erlang 24.3.4
-ruby 2.6.6
```
Comment (DavideSilva) [inline .tool-versions:4]: We still use `foreman` to run the `bin/setup´ script. While it's true that any version of Ruby should work for this, I'd like to have the Ruby dependency explicit. What do you think?

Comment (jfranciscosousa) [reply .tool-versions:4]: Sure no problem. It's probably faster to install `foreman` from `brew` or `apt-get` though. (ruby build takes ages)

Comment (DavideSilva) [reply .tool-versions:4]: Yeah, I agree. We should probably find a different solution for using `foreman` just to start the local development environment. It's very convenient but might be a pain for the first setup of the repo.

For now, let's keep this and I'll come up with something on a later PR

**Review by DavideSilva — APPROVED** [review]
I tagged @davelange and @paulacarneiro just so that they can take a look into these new changes of svelte-kit but seems good to me 

:shipit:

**Review by DavideSilva — CHANGES_REQUESTED** [review]
Deploy preview is broken

**Comment by jfranciscosousa** [comment]
@DavideSilva made a tweak to the Netlify config, please approve the build just to see if it fixes it.

**Comment by jfranciscosousa** [comment]
@DavideSilva the latest changes work on my own Netlify deploy (I made one from my fork). I think the `esbuild` option does the trick. Let me know how it goes.

**Comment by DavideSilva** [comment]
@jfranciscosousa "Something exploded"

This might be because the Heroku preview app was not deployed. I triggered the deploy manually to see if that was the issue

**Comment by DavideSilva** [comment]
Yup, that was the problem, it works now

:shipit:

**Review by DavideSilva — APPROVED** [review]

**Pull request merged**